scripts/pdf_utils.py

```
@@ -271,18 +272,246 @@ def pdf_to_docx(input_path: str, output_dir: str, password: str = None) -> str:
-    
-    # Handle encrypted PDFs
+
     decrypted_path, needs_cleanup = _get_decrypted_pdf_path(input_path, password)
-    
+
     try:
         cv = Converter(decrypted_path)
-        cv.convert(str(output_file))
+        try:
+            cv.convert(str(output_file), multi_processing=True)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)